pkg/ingester/flush.go

```diff
@@ -349,6 +357,10 @@ func (i *Ingester) flushChunks(ctx context.Context, fp model.Fingerprint, metric
 	// Record statistsics only when actual put request did not return error.
 	for _, chunkDesc := range chunkDescs {
 		utilization, length, size := chunkDesc.C.Utilization(), chunkDesc.C.Len(), chunkDesc.C.Size()
+		if length < i.cfg.MinChunkLength {
+			droppedChunks.Inc()
+			continue
```

Comment:
I'd want these passed to `util.Event()` for analysis.

Reply:
:+1: